<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maint</class>
 <widget class="QDialog" name="Maint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero Computer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Typeof maintenance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de debut</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de fin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>123</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>descrp de maintenance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>253</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>253</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>prevenntaire</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>253</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>corrective</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>253</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>253</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>253</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre en maintenance</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
